pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "envier~=0.5",
```

Comment:
Can't comment on the diff on the ddsketch line, but I believe ddsketch has protobuf dependency as well that we'll need to deal with: https://github.com/DataDog/sketches-py/blob/c0b33bcf2dadbd0b16ee44cecfcceae86e93fd6c/setup.py#L23-L24. ddsketch is only used when tracer computed stats is enabled.

Reply:
https://github.com/DataDog/sketches-py/pull/62